### ISSUE TITLE
Deselecting a selected entry while using custom sort doesn't work well

### ISSUE THREAD

**Issue by alexbakker** [beemdevelopment/Aegis#1002]
Steps to reproduce:
- Have a couple of entries in your entry list
- Enable "Custom" sorting
- Long press an entry to select it
- Try to deselect the entry by tapping it and notice that it doesn't work most of the time

I can't reproduce this in an emulator. This only happens on a physical device when actually tapping the screen to deselect an entry. I suspect this has something to do with ``SimpleItemTouchHelperCallback.isLongPressDragEnabled()`` returning ``true`` and a tiny "drag" movement being detected when tapping the an entry.

**Comment by codeall9** [comment]
yah, the code below won't work since human fingers are different from computer cursor

``` java
// EntryAdapter.java
entryHolder.itemView.setOnClickListener(/* ... */)
entryHolder.itemView.setOnTouchListener(new View.OnTouchListener() {
    @Override
    public boolean onTouch(View v, MotionEvent event) {
        // Start drag if this is the only item selected
        if (event.getActionMasked() == MotionEvent.ACTION_MOVE
                && isEntryDraggable(entryHolder.getEntry())) {
            _view.startDrag(entryHolder);
            return true;
        }
        return false;
    }
});
```

The general solution is to use drag handle to drag the entry instead, so it can avoid conflict with click event
``` java
entryHolder.itemView.setOnClickListener(/* ... */)
entryHolder.dragHandle.setOnTouchListener(/* ... */)
```

@alexbakker 
I can help to fix this if you don't have any concerns

**Comment by alexbakker** [comment]
@codeall9 I haven't looked into this much, but I feel like there has to be a way to detect tiny drag movements and ignore those. Apparently this is called "slop": https://developer.android.com/develop/ui/views/touch-and-input/gestures/viewgroup.

**Comment by codeall9** [comment]
indeed, we can detect the tiny movement, but it also increase complexity of the `EntryAdapter` depending on how many gestures or view modes need to be supported, ex: nested scroll, fling, titles mode

I believe most application show a drag handle instead and simply solve the conflict

@alexbakker 
let me know what do you prefer
